src/Symfony/Component/Scheduler/Progress/ProgressInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Scheduler\Progress;
+
+interface ProgressInterface
```

Comment:
I'm wondering is `StateInterface` would be a better name?

Reply:
Are you sure? The locking feature doesn't match with a "state". It's more like a point on a schedule or a timeline that can be accessible or not. I've tried: state, position, cursor, progress... Feels like walking in circles. I am not good at it. Thank you for your help :slightly_smiling_face: